src/os/bluestore/simple_bitmap.h

```diff
@@ -0,0 +1,139 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Author: Gabriel BenHanokh <gbenhano@redhat.com>
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+#pragma once
+#include <cstdint>
+#include <iostream>
+#include <string>
+#include <cstring>
+#include <cmath>
+#include <iomanip>
+
+#include "include/ceph_assert.h"
+
+struct extent_t {
+  uint64_t offset;
+  uint64_t length;
+};
+
+class SimpleBitmap {
+public:
+  //----------------------------------------------------------------------------
+  friend std::ostream& operator<<(std::ostream& os, const SimpleBitmap& bmap);
+
+  //----------------------------------------------------------------------------
+  //throw bad_alloc
+  SimpleBitmap(CephContext *_cct, const std::string &_path, uint64_t num_bits);
+
```

Comment:
We are manually maintaining m_arr, we should either implement, or delete (preferred option) copy constructor.